controllers/tasks.js

```diff
@@ -43,6 +43,33 @@ const fetchTasks = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
+
+/**
+ * Fetches all the tasks of the requested user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getUserTasks = async (req, res) => {
+  try {
+    const { username } = req.params
+    const allTasks = await tasks.fetchUserTasks(username)
+
+    if (allTasks === false) {
+      return res.boom.notFound('User doesn\'t exist')
+    }
```

Comment:
This doesn't make much sense to me. Like how can `allTasks` be false? 😅 

More on this in `fetchUserTasks` function